### PR TITLE
feat: add next birthday handler

### PR DESCRIPTION
**Pull request by norbinsh**
Addresses #1

### CODE REVIEW

File: commands/birthdays/next_birthday.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+        c = calculate_dates(birthday['date'], now)
+        if c < distance:
+            distance = c
+            nearest_birthday = birthday
+    if not nearest_birthday:
+        return send_message(context, chat_id, "Please load birthdays first.")
+    message = f"Next birthday is in {c} day(s): {parse_birthday(nearest_birthday)}"
+    return send_message(context, chat_id, message)
+
+def calculate_dates(original_date, now):
```
Comment (DanTcheche) [inline commands/birthdays/next_birthday.py]: I would rename this to get_difference_between_dates_in_days(original_date, date_to_compare).

File: commands/birthdays/next_birthday.py [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+    db_connector, birthdays_col = db_handler('birthday')
+
+    query = {"chat_id": chat_id}
+    next_birthday = db_connector.search_records(birthdays_col, query)
+
+    distance = 365
+    nearest_birthday = None
+    now = datetime.now()
+    for birthday in next_birthday:
+        c = calculate_dates(birthday['date'], now)
```
Comment (DanTcheche) [inline commands/birthdays/next_birthday.py]: I would name this variable something more comprehensible, like difference_in_days.

**Review by DanTcheche — CHANGES_REQUESTED** [review]

File: commands/birthdays/next_birthday.py (hunk 2 above)
Comment (norbinsh) [reply commands/birthdays/next_birthday.py]: done

File: commands/birthdays/next_birthday.py (hunk 1 above)
Comment (norbinsh) [reply commands/birthdays/next_birthday.py]: done

**Pull request merged**